### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by shrutivanw**
CS Fundamentals' assignment to compute the nth fibonacci number in the series starting with 0.

### CODE REVIEW

File: lib/fibonacci.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Computes the nth fibonacci number in the series starting with 0.
+# fibonacci series: 0 1 1 2 3 5 8 13 21 ...
+# e.g. 0th fibonacci number is 0
+# e.g. 1st fibonacci number is 1
+# ....
+# e.g. 6th fibonacci number is 8
+def fibonacci(n)
+  puts "NOT IMPLEMENTED"
+  return n
```
Comment (Hamled) [inline lib/fibonacci.rb]: How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console for code being tested, because that output kind of gets randomly interspersed with the test runner's output.

File: specs/fibonacci_spec.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/fibonacci'
+
+describe "fibonacci" do
+  describe "basic tests" do
+    describe "test 1" do
```
Comment (Hamled) [inline specs/fibonacci_spec.rb]: Same as the other repos, we should probably remove the `describe` blocks that are immediate parents of individual tests.

File: specs/fibonacci_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    describe "test 5" do
+      it "fibonacci of 6" do
+        fibonacci(6).must_equal 8
+      end
+    end
+  end
+
+  # if the parameter is an object, check for nil
+  describe "nil object input test" do
+    it "nil object is not an integer" do
```
Comment (Hamled) [inline specs/fibonacci_spec.rb]: I think we could move this nil input test case into the edge cases section.

File: README.md [hunk 4]
```diff
@@ -1 +1,11 @@
-# fibonacci
\ No newline at end of file
+# Fibonacci
+Compute the nth fibonacci number.
```
Comment (Hamled) [inline README.md]: It'd probably be helpful to link to the [Wikipedia article for Fibonacci number](https://en.wikipedia.org/wiki/Fibonacci_number).

File: lib/fibonacci.rb (hunk 1 above)
Comment (shrutivanw) [reply lib/fibonacci.rb]: Done!

File: specs/fibonacci_spec.rb (hunk 2 above)
Comment (shrutivanw) [reply specs/fibonacci_spec.rb]: Done!

File: specs/fibonacci_spec.rb (hunk 3 above)
Comment (shrutivanw) [reply specs/fibonacci_spec.rb]: Done!

File: README.md (hunk 4 above)
Comment (shrutivanw) [reply README.md]: done!

**Comment by shrutivanw** [comment]
Feedback incorporated. Thank you @Hamled
